buddypress/members/single/parts/item-subnav.php

```diff
@@ -16,8 +16,8 @@
 	}
 ?>
 <div class="nhsuk-grid-full-width">
-    <nav class="nhsuk-bordered-tabs-container secondary" id="subnav" role="navigation" aria-label="<?php esc_attr_e( 'User administration menu', 'nightingale' ); ?>">
-        <ul class="nhsuk-bordered-tabs">
+    <nav style="clear: left;" class="nhsuk-bordered-tabs-container secondary" id="subnav" role="navigation" aria-label="<?php esc_attr_e( 'User administration menu', 'nightingale' ); ?>">
```

Comment:
Hardcoding style for nav may break some other media output eg: mobile and print. At minimum we can add clearing to CSS. If anything we should try to utilise the existing clearfix functions eg: https://github.com/nhsuk/nhsuk-frontend/blob/main/packages/core/README.md#clearfix